### ISSUE TITLE
Documentation

### ISSUE THREAD

**Issue by VeeeneX** [lord2800/convey#3]
Amazing code but please, write understandable documentation :+1:

**Comment by lord2800** [comment]
Yeah, it's on my list of things do to (need to find a better way to write docs for this and a few other projects too). Thanks for opening this--I'll keep it for tracking.

**Comment by lord2800** [comment]
Just letting you know, this is still on my todo list. I've been crazy busy at work though, so I haven't gotten to it yet. :(

**Comment by VeeeneX** [comment]
It's not necessary, it's just a tune :smile:
